lib/src/screen/moor_table_content_list_screen.dart

```diff
@@ -152,6 +158,9 @@ class _MoorTableContentListScreenState extends State<MoorTableContentListScreen>
       content: Text(message),
       behavior: SnackBarBehavior.floating,
     );
-    _key.currentState.showSnackBar(snackBar);
+
+    if (_key.currentContext != null) {
```

Comment:
move this to a seperate local variable. So you don't have to do the null check twice